e2e-app/tsconfig.json

```diff
@@ -6,23 +6,13 @@
     "paths": {
       "@ng-bootstrap/ng-bootstrap": [
         "../src/index"
-      ],
-      "@test": [
-        "./playwright/index"
       ]
     }
   },
   "files": [
     "src/main.ts",
     "src/polyfills.ts"
   ],
-  "include": [
-    "src/**/*.d.ts"
-  ],
-  "exclude": [
-    "src/**/*.po.ts",
-    "src/**/*.e2e-spec.ts"
-  ],
```

Comment:
These changes are fine, but there other tsconfig where po.ts and e2e-spec.ts are referenced. Should we remove them too ?